contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -139,7 +139,7 @@ contract WithdrawalQueue {
     uint256 public finalizedRequestsCounter = 0;
 
     /// @notice queue for withdrawal requests
-    WithdrawalRequest[] public queue;
+    WithdrawalRequest[] internal queue;
```

Comment:
why internal?

Reply:
No need to expose internal cumulative nature of the queue to users. There are more convenient methods, like `getWithdrawalRequestStatus` and others.